src/NUnitFramework/nunitlite.tests/nunitlite.tests-2.0.csproj

```diff
@@ -82,6 +82,10 @@
     <Compile Include="TokenizerTests.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="NUnit.System.Linq, Version=0.6.0.0, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\NUnit.System.Linq.0.6.0\lib\net20\NUnit.System.Linq.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
```

Comment:
This may be a leftover change from when all the other projects was in the solution.  I was getting a build error that said I needed to add the package in some projects, though.